Gemfile.lock

```diff
@@ -131,7 +131,9 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.10)
```

Comment:
I'd say we could dump this specific update and just update rails to the newest version, since it dropped mimemagic.

See [here](https://weblog.rubyonrails.org/2021/3/26/marcel-upgrade-releases/).